legacy/src/app/controllers/node_details.js

```diff
@@ -1759,6 +1759,12 @@ function NodeDetailsController(
     if ($scope.isDevice && activeNode) {
       $scope.ip_assignment = activeNode.ip_assignment;
     }
+    // Handle updating the area when the browser navigates back/forward.
+    $scope.$on("$locationChangeStart", () => {
+      $scope.section.area = angular.isString($location.search().area)
```

Comment:
Just to be safe I think we should add a guard on `$location.search(). I feel like accessing properties of an undefined object is one of the most common issues with the angular codebase.

Reply:
Good idea. Done.